pkg/cmd/repo/create/create_test.go

```diff
@@ -163,25 +153,11 @@ func TestRepoCreate_outsideGitWorkDir(t *testing.T) {
 
 	httpClient := &http.Client{Transport: reg}
 
-	var seenCmds []*exec.Cmd
-	cmdOutputs := []test.OutputStub{
-		{
-			Error: errors.New("Not a git repository"),
-		},
-		{},
-		{},
-	}
-	//nolint:staticcheck // SA1019 TODO: rewrite to use run.Stub
-	restoreCmd := run.SetPrepareCmd(func(cmd *exec.Cmd) run.Runnable {
-		if len(cmdOutputs) == 0 {
-			t.Fatal("Too many calls to git command")
-		}
-		out := cmdOutputs[0]
-		cmdOutputs = cmdOutputs[1:]
-		seenCmds = append(seenCmds, cmd)
-		return &out
-	})
-	defer restoreCmd()
+	cs, cmdTeardown := run.Stub()
+	defer cmdTeardown(t)
+
+	cs.Register(`git remote add -f origin https://github\.com/OWNER/REPO\.git`, 0, "")
+	cs.Register(`git rev-parse --show-toplevel`, 0, "")
```

Comment:
Should this command exit with nonzero status, since we're emulating not being in a git repo?